articles/openshift/oshift-hawkular-metrics.md

```diff
@@ -0,0 +1,75 @@
+---
+title: How to use the Hawkular Metrics API | UKCloud Ltd
+description: Shows how to structure basic queries to the Hawkular Metrics API in order to get resource statistics back
+services: openshift
+author: Kieran O'Neill
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Hawkular Metrics
+toc_fullpath: How To/oshift-how-create-users.md
+toc_mdlink: oshift-hawkular-metrics.md
+---
+
+# How to use the Hawkular Metrics API
+
+## Overview
+
+This guide is intended to give you enough information to forumulate basic memory usage or cpu queries against nodes in your cluster. For more advanced usage please refer to the Hawkular documentation at: https://www.hawkular.org/hawkular-metrics/docs/user-guide/#_metrics or the OpenShift Hawkular repository readme at https://github.com/openshift/origin-metrics/blob/master/docs/hawkular_metrics.adoc
+
+## Using the API
+
+1. Firstly, you must ensure that the user you will be using to query the API has reader permissions for the corresponding OpenShift project to the Hawkular-Tenant you are querying. If you are querying the _system tenant then the user must have cluster-reader permissions.
+
+2. To get a list of all node level counters and gauges you can use you would run the following:
```

Comment:
Change to:

To get a list of all node level counters and gauges you can use, run the following: